src/Export/ExportService.php

```diff
@@ -206,20 +207,26 @@ public function allRegistrationDataToCSV(Event $event, User $adminUser): array
         ];
 
         foreach ($participants as $participant) {
-            if ($participant instanceof PatrolLeader) {
-                $pPart = [
+            if ($participant instanceof PatrolLeader || $participant instanceof TroopLeader) {
+                $ptPart = [
                     (string)$participant->id,
                     $participant->patrolName ?? '',
                     (string)$participant->getPatrolParticipantsCount(),
```

Comment:
tady je třeba rozlišit, jestli je to TroopLeader nebo PatrolLeader. Ono se to bude do budoucna štěpit ještě víc (třeba $participant->patrolName bude ->troopName), tak prosím o rozdělení toho ifu -> zanechat `$participant instanceof PatrolLeaer` a přidat další elseif `$participant instanceof TroopLeader`